terraform/alerting/alerts.tf

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+locals {
+  slow_services = format("(%s)", join("|", [
+    "export",
```

Comment:
also needs cleanup services

Reply:
export service actually rarely exceeds 10s at 99 percentile.

It looks like based on the last 1 month of data, only cleanup-exposure and cleanup-export should have 20s of alerting threshold.

![image](https://user-images.githubusercontent.com/146133/98910627-4a877980-2478-11eb-8659-fc3b1aac34f7.png)